src/sentry/search/events/builder/errors.py

```diff
@@ -30,7 +31,7 @@
 value_converters = {"status": convert_status_value}
 
 
-class ErrorsQueryBuilderMixin:
+class ErrorsQueryBuilderMixin(BaseQueryBuilder):
```

Comment:
The missing parameters are set here:
https://github.com/getsentry/sentry/blob/b5f1809d0ea8dcfc6cbb4f61c01c586ca533f801/src/sentry/search/events/builder/base.py#L180-L289

```
mypy src/sentry/search/events/builder/errors.py
src/sentry/search/events/builder/errors.py:40: error: "parse_query" undefined in superclass  [misc]
src/sentry/search/events/builder/errors.py:43: error: "ErrorsQueryBuilderMixin" has no attribute "params"  [attr-defined]
src/sentry/search/events/builder/errors.py:44: error: "ErrorsQueryBuilderMixin" has no attribute "params"  [attr-defined]
src/sentry/search/events/builder/errors.py:45: error: "ErrorsQueryBuilderMixin" has no attribute "params"  [attr-defined]
src/sentry/search/events/builder/errors.py:52: error: "ErrorsQueryBuilderMixin" has no attribute "dataset"  [attr-defined]
src/sentry/search/events/builder/errors.py:52: error: "ErrorsQueryBuilderMixin" has no attribute "sample_rate"  [attr-defined]
src/sentry/search/events/builder/errors.py:62: error: "resolve_params" undefined in superclass  [misc]
src/sentry/search/events/builder/errors.py:68: error: "ErrorsQueryBuilderMixin" has no attribute "params"  [attr-defined]
src/sentry/search/events/builder/errors.py:81: error: "resolve_query" undefined in superclass  [misc]
src/sentry/search/events/builder/errors.py:85: error: "aliased_column" undefined in superclass  [misc]
src/sentry/search/events/builder/errors.py:100: error: "ErrorsQueryBuilderMixin" has no attribute "resolve_column_name"  [attr-defined]
src/sentry/search/events/builder/errors.py:108: error: "ErrorsQueryBuilderMixin" has no attribute "dataset"  [attr-defined]
Found 12 errors in 1 file (checked 1 source file)
```

Reply:
mixins in general aren't typesafe -- this should probably be refactored to composition or otherwise